com.lakoliu.Furtherance.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "com.lakoliu.Furtherance",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "furtherance",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=session-bus"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
this is not allowed, there is no network during the build. you need to use flatpak-builder-tools to generate a list of crates to download: https://github.com/flatpak/flatpak-builder-tools